src/release/main.d

```diff
@@ -335,6 +332,28 @@ ActionList prepareMinorRelease ( ref HTTPConnection con, ref Repository repo,
 
 /*******************************************************************************
 
+    Prepares the actions to clar the release notes
+
+    Returns:
+        actions required to clear the release notes
+
+*******************************************************************************/
+
+ActionList clearReleaseNotes ( )
```

Comment:
I suppose it depends how we want the release notes to look. If we want the notes for subsequent majors to just say "Also includes merge of vx.y.0 into va.b.x", then that would be the way to do it.

That approach sounds good to me.